scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -27,7 +27,7 @@
 from .process import set_column_map
 
 # These directories are excluded from testing
-_EXCLUDE_DIRS = ['common', 's2201', '__pycache__']
+_EXCLUDE_DIRS = ['common', 's2201', '__pycache__', '~']
```

Comment:
is there a '~' directory in this path or does the os util to list dirs pull in home dir as well?

Reply:
no, macOS lists home directory along with `.` and `..` in the list of directories. If the test is run on macOS, it will fail.